.github/workflows/refapp-2x-legacy.yml

```diff
@@ -0,0 +1,37 @@
+name: RefApp 2.x Legacy Selenium Firefox
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: run db and web containers
+        run: |
+          docker-compose -f docker/docker-compose-refqa.yml up -d
+      - name: wait for openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/openmrs/login.htm)" != "200" ]]; do sleep 1; done
+      - name: Run qaframework on selenium
```

Comment:
@kdaud  still am not understanding what this workflow triggers, to me it seems to be the working as this [workflow ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/.github/workflows/qa.yml), is there a difference. Are we really triggering selenium or we can simply handle them without triggering them as along as they dont contradict, I mean simply doing the way you have done but without this workflow because it seems not to be working as expected@kdaud  still am not understanding what this workflow triggers, to me it seems to be the working as this [workflow ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/.github/workflows/qa.yml), is there a difference. Are we really triggering selenium or we can simply handle them without triggering them as along as they dont contradict, I mean simply doing the way you have done but without this workflow because it seems not to be working as expected

Reply:
@sherrif10 I agree!

Reply:
Now that we have the BDD workflow running independently as we intended them to be running!